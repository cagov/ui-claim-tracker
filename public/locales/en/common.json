{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "EDD Employment Development Department State of California Home",
    "edd-home": "EDD Home",
    "help": "Help",
    "welcome": "Welcome, name",
    "logout": "Log out",
    "uio-home": "UI Online Home"
  },
  "title": "Claim Tracker",
  "welcome": "Claim Tracker",
  "change-locale": "En español",
  "claim-status": {
    "title": "Claim Status",
    "pending-determination": "We need to confirm your eligibility for benefits.",
    "generic-pending": "Your payments are pending further review.",
    "generic-all-clear": "There are no pending issues at this time.",
    "next-steps": "Next Steps"
  },
  "claim-details": {
    "title": "Claim Details",
    "benefit-year": "Benefit Year",
    "program-type": "Program Type",
    "claim-balance": "Claim Balance",
    "weekly-benefit-amount": "Weekly Benefit Amount",
    "last-payment-issued": "Last Payment Issued",
    "extension-type": "Extension Type",
    "extension-end-date": "Extension End Date"
  },
  "work-in-progress": {
    "title": "This page is a work in progress.",
    "message": "While we test this new part of UI Online, the information here may be limited for your claim. We're working to add information about common issues that lead to <1>pending payments</1>."
  },
  "footer": {
    "toTop": "Back to Top",
    "contact": "Contact EDD",
    "conditionsOfUse": "Conditions of Use",
    "privacyPolicy": "Privacy Policy",
    "accessibility": "Accessibility",
    "copyright": "Copyright © 2021 State of California"
  },
  "timeout-modal": {
    "header": "Your Session Will End Soon",
    "warning": "To protect your information, you will be logged out in {{numberOfMinutes}} minutes if you do not continue.",
    "button": "Stay Logged In"
  }
}
