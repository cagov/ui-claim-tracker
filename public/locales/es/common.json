{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "Página principal del Departamento de Desarrollo de Empleo del EDD",
    "edd-home": "EDD Casa",
    "help": "Ayuda",
    "welcome": "Bienvenidas, nombre",
    "logout": "Cerrar sesión",
    "uio-home": "UI Online Casa"
  },
  "title": "Claim Tracker",
  "welcome": "Claim Tracker",
  "change-locale": "In English",
  "claim-status": {
    "title": "Claim Status",
    "next-steps": "Próximos Pasos"
  },
  "claim-details": {
    "title": "Claim Details",
    "benefit-year": "Año de beneficio",
    "program-type": "Tipo de Programa",
    "claim-balance": "Balance de Reclamación",
    "weekly-benefit-amount": "Monto de Beneficio Semanal",
    "last-payment-issued": "Último Pago Emitido",
    "extension-type": "Tipo de Extensión",
    "extension-end-date": "Fecha de Finalización de la Extensión"
  },
  "work-in-progress": {
    "title": "Esta página es un trabajo en progreso.",
    "message": "Mientras probamos esta nueva parte de UI Online, la información aquí puede ser limitada para su reclamo. Estamos trabajando para agregar información sobre problemas comunes que generan <1>pagos pendientes</1>."
  },
  "footer": {
    "toTop": "Volver Arriba",
    "contact": "Contacta con EDD",
    "conditionsOfUse": "Condiciones de Uso",
    "privacyPolicy": "Política de Privacidad",
    "accessibility": "Accesibilidad",
    "copyright": "Copyright © 2021 Estado de California"
  },
  "timeout-modal": {
    "header": "Tu sesión terminará pronto",
    "warning": "Para proteger su información, se cerrará la sesión en {{numberOfMinutes}} minutos si no continúa.",
    "button": "Sigue Trabajando"
  }
}
