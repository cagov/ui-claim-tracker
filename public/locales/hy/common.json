{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "EDD՝ Կալիֆոռնիա նահանգի Զբաղվածության զարգացման վարչության Գլխավոր էջ",
    "edd-home": "EDD գլխավոր էջ",
    "help": "Օգնություն",
    "logout": "Դուրս գալ",
    "uio-home": "UI Գլխավոր էջ",
    "uio-certify": "Հավաստագրել",
    "uio-payments": "Վճարումներ",
    "uio-history": "Պատմաշարք",
    "uio-status": "Կարգավիճակ",
    "uio-profile": "Պրոֆիլ",
    "uio-inbox": "Փոստարկղ",
    "uio-contact": "Կապ"
  },
  "title": "Հայտի կարգավիճակի հետագծող",
  "welcome": "Հայտի կարգավիճակի հետագծող",
  "errors": {
    "not-found": {
      "label": "Հնարավոր չէ գտնել այդ էջը:",
      "entry": "Ձեր ընտրած հղման էջը չի աշխատում կամ հեռացված է:"
    },
    "button": "Վերադառնալ UI Գլխավոր էջ",
    "server-error": {
      "label": "Մեր կողմից սխալ կա։",
      "entry": "Ձեր խնդրանքը հնարավոր չէ կատարել:"
    }
  },
  "maintenance": {
    "label": "Մենք շուտով կվերադառնանք",
    "entry": "Այս էջը ներկայումս անհասանելի է համակարգի սպասարկման ընթացքում:",
    "entry2": "Շնորհակալություն ձեր համբերության համար։ Մենք շարունակում ենք բարելավել մեր ծառայությունները:"
  },
  "claim-status": {
    "title": "Հայտի կարգավիճակ",
    "your-next-steps": "Ձեր հաջորդ քայլերը",
    "edd-next-steps": "EDD-ի հաջորդ քայլերը"
  },
  "claim-details": {
    "title": "Հայտի մանրամասներ",
    "benefit-year": "նպաստային տարի",
    "program-type": "Հայտի տեսակ",
    "claim-balance": "Հայտի հաշվեկշիռ",
    "weekly-benefit-amount": "Շաբաթական նպաստի գումար",
    "last-payment-issued": "Վերջին վճարումը կատարվել է",
    "none": "Ոչինչ",
    "extension-type": "Ընդլայնման տեսակ"
  },
  "footer": {
    "toTop": "Վերադառնալ սկիզբ",
    "contact": "Կապվեք EDD-ի հետ",
    "conditionsOfUse": "Օգտագործման պայմաններ",
    "privacyPolicy": "Գաղտնիության քաղաքականություն",
    "accessibility": "Մատչելիություն",
    "copyright": "Copyright © 2021 State of California"
  },
  "timeout-modal": {
    "header": "Ձեր նիստը  շուտով կավարտվի",
    "warning": "Ձեր տվյալները պաշտպանելու համար՝ դուք դուրս կքաք {{count}} րոպեից, եթե չեք ցանկանում շարունակել:",
    "warning_plural": "Ձեր տվյալները պաշտպանելու համար՝ դուք դուրս կգրվեք {{count}} րոպեից, եթե չեք ցանկանում շարունակել:",
    "button": "Վերադառնալ UI Գլխավոր էջ"
  },
  "time": {
    "between-range": ", Խաղաղօվկիանոսյան ժամանակով {{range}} {{ampm}}-ի միջև",
    "between-start-end": ", Խաղաղօվկիանոսյան ժամանակով {{start.time}} {{start.ampm}} և {{end.time}} {{end.ampm}} միջև",
    "am": "ցերեկը",
    "pm": "գիշերը"
  },
  "urls": {
    "edd": {
      "ui-certify": "https://edd.ca.gov/Unemployment/certify.htm",
      "ui-claim-status": "https://www.edd.ca.gov/unemployment/claim-status.htm",
      "ui-reopen": "https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm",
      "ui-calculator": "https://edd.ca.gov/Unemployment/UI-Calculator.htm"
    },
    "hypertext": {
      "uio-homepage": "UI Online-ի Գլխավոր էջ",
      "pending-payment": "Հայտի կարգավիճակ՝ առկախ Վճարում",
      "certify": "Հավաստագրել նպաստների համար",
      "reopen": "Վերաբացեք ձեր հայտը",
      "reapply": "Կրկին դիմել գործազրկության նպաստի համար",
      "enough-wages": "բավարար աշխատավարձ"
    }
  },
  "external-navigation-modal": {
    "header": "Հաստատեք հղումը",
    "warning": "Դուք հավելվածից դուրս նոր պատուհան եք բացում: Ընտրեք Շարունակել՝ նոր պատուհան բացելու համար կամ ընտրեք Չեղարկել՝ հավելված վերադառնալու համար:",
    "cancel-button": "Չեղարկել",
    "continue-button": "Շարունակել"
  }
}
