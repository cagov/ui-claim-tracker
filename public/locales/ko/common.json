{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "EDD 캘리포니아주 고용개발부 홈",
    "edd-home": "EDD 홈",
    "help": "도움말",
    "logout": "로그아웃",
    "uio-home": "실업 보험 홈",
    "uio-certify": "인증",
    "uio-payments": "지급",
    "uio-history": "기록",
    "uio-status": "상태",
    "uio-profile": "프로필",
    "uio-inbox": "메시지함",
    "uio-contact": "문의"
  },
  "title": "청구 상태 추적기",
  "welcome": "청구 상태 추적기",
  "errors": {
    "not-found": {
      "label": "해당 페이지를 찾을 수 없습니다.",
      "entry": "귀하가 클릭한 링크가 손상되었거나 페이지가 제거되었을 수 있습니다."
    },
    "button": "실업 보험 홈으로 돌아가기",
    "server-error": {
      "label": "EDD에서 오류가 발생했습니다.",
      "entry": "귀하의 요청을 처리할 수 없습니다."
    }
  },
  "maintenance": {
    "label": "곧 서비스를 재개하겠습니다",
    "entry": "현재 시스템 점검 중이어서 이 페이지를 사용할 수 없습니다.",
    "entry2": "서비스를 개선하는 동안 양해해 주셔서 감사합니다."
  },
  "claim-status": {
    "title": "청구 상태",
    "your-next-steps": "다음 단계",
    "edd-next-steps": "EDD 다음 단계"
  },
  "claim-details": {
    "title": "청구 상세 정보",
    "benefit-year": "급여 연도",
    "program-type": "청구 유형",
    "claim-balance": "청구 잔액",
    "weekly-benefit-amount": "주간 급여 액수",
    "last-payment-issued": "지급된 마지막 급여",
    "none": "없음",
    "extension-type": "연장 유형"
  },
  "footer": {
    "toTop": "위로 올라가기",
    "contact": "EDD에 문의하기",
    "conditionsOfUse": "사용 조건",
    "privacyPolicy": "개인정보 보호정책",
    "accessibility": "접근성",
    "copyright": "Copyright © 2021 State of California"
  },
  "timeout-modal": {
    "header": "세션이 곧 종료됩니다",
    "warning": "활동이 없으면 정보를 보호하기 위해 {{count}}분 후에 로그아웃됩니다.",
    "warning_plural": "활동이 없으면 정보를 보호하기 위해 {{count}}분 후에 로그아웃됩니다.",
    "button": "실업 보험 홈으로 돌아가기"
  },
  "time": {
    "between-range": ", {{ampm}} {{range}}시(태평양 시간)",
    "between-start-end": ", {{start.ampm}} {{start.time}}시 ~ {{end.ampm}} {{end.time}}시(태평양 시간)",
    "am": "오전",
    "pm": "오후"
  },
  "urls": {
    "edd": {
      "ui-certify": "https://edd.ca.gov/Unemployment/certify.htm",
      "ui-claim-status": "https://www.edd.ca.gov/unemployment/claim-status.htm",
      "ui-reopen": "https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm",
      "ui-calculator": "https://edd.ca.gov/Unemployment/UI-Calculator.htm"
    },
    "hypertext": {
      "uio-homepage": "실업 보험 온라인 홈페이지",
      "pending-payment": "청구 상태: 보류 중인 지급",
      "certify": "급여에 대한 인증",
      "reopen": "청구 재개",
      "reapply": "실업 급여 재신청",
      "enough-wages": "충분한 임금"
    }
  },
  "external-navigation-modal": {
    "header": "탐색 확인",
    "warning": "애플리케이션 밖에서 새 탭을 열고 있습니다. 계속을 클릭하여 새 탭을 열거나 취소를 선택하여 애플리케이션으로 돌아가세요.",
    "cancel-button": "취소",
    "continue-button": "계속"
  }
}
