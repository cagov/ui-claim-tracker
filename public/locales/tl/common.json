{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "Home ng Departamento ng Pag-unlad sa Trabaho (EDD) ng Estado ng California",
    "edd-home": "Home ng EDD",
    "help": "Tulong",
    "logout": "Mag-log Out",
    "uio-home": "Home ng UI",
    "uio-certify": "Mag-certify",
    "uio-payments": "Aktibidad sa Pagbabayad",
    "uio-history": "History",
    "uio-status": "Status",
    "uio-profile": "Profile",
    "uio-inbox": "Inbox",
    "uio-contact": "Makipag-ugnayan"
  },
  "title": "Tracker ng Status ng Claim",
  "welcome": "Tracker ng Status ng Claim",
  "errors": {
    "not-found": {
      "label": "Hindi namin mahanap ang page na iyan.",
      "entry": "Maaaring sira ang link na na-click mo o tinanggal na ang page na ito."
    },
    "button": "Bumalik sa Home ng UI",
    "server-error": {
      "label": "Nagkaroon ng error sa panig namin.",
      "entry": "Hindi namin maproseso ang iyong request."
    }
  },
  "maintenance": {
    "label": "Babalik Kami Sa Lalong Madaling Panahon",
    "entry": "Kasalukuyang hindi available ang page na ito habang isinasagawa ang pagmementina ng system.",
    "entry2": "Salamat sa iyong pasensya habang patuloy naming pinaghuhusay ang aming mga serbisyo."
  },
  "claim-status": {
    "title": "Status ng Claim",
    "your-next-steps": "Ang Mga Susunod Mong Gagawin",
    "edd-next-steps": "Mga Susunod na Hakbang ng EDD"
  },
  "claim-details": {
    "title": "Mga Detalye ng Claim",
    "benefit-year": "Taon ng Benepisyo",
    "program-type": "Uri ng Claim",
    "claim-balance": "Balanse ng Claim",
    "weekly-benefit-amount": "Halaga ng Lingguhang Benepisyo",
    "last-payment-issued": "Huling Inisyung Bayad",
    "none": "Wala",
    "extension-type": "Uri ng Pagpapalawig"
  },
  "footer": {
    "toTop": "Bumalik sa Itaas",
    "contact": "Makipag-ugnayan sa EDD",
    "conditionsOfUse": "Mga Kondisyon ng Paggamit",
    "privacyPolicy": "Patakaran sa Privacy",
    "accessibility": "Accessibility",
    "copyright": "Copyright Â© 2021 State of California"
  },
  "timeout-modal": {
    "header": "Malapit nang Matapos ang Iyong Session",
    "warning": "Para maprotektahan ang iyong impormasyon, mala-log out ka sa loob ng {{count}} minuto kung hindi ka magpapatuloy.",
    "warning_plural": "Para maprotektahan ang iyong impormasyon, mala-log out ka sa loob ng {{count}} minuto kung hindi ka magpapatuloy.",
    "button": "Bumalik sa Home ng UI"
  },
  "time": {
    "between-range": ", sa pagitan ng {{range}} {{ampm}} Pacific time",
    "between-start-end": ", sa pagitan ng {{start.time}} {{start.ampm}} at {{end.time}} {{end.ampm}} Pacific time",
    "am": "a.m.",
    "pm": "p.m."
  },
  "urls": {
    "edd": {
      "ui-certify": "https://edd.ca.gov/Unemployment/certify.htm",
      "ui-claim-status": "https://www.edd.ca.gov/unemployment/claim-status.htm",
      "ui-reopen": "https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm",
      "ui-calculator": "https://edd.ca.gov/Unemployment/UI-Calculator.htm"
    },
    "hypertext": {
      "uio-homepage": "Homepage ng UI Online",
      "pending-payment": "Status ng Claim: Nakabinbin ang Pagbabayad",
      "certify": "i-certify para sa mga benepisyo",
      "reopen": "muling buksan ang iyong claim",
      "reapply": "Mag-apply muli para sa mga benepisyo para sa pagkawala ng trabaho",
      "enough-wages": "mga sapat na suweldo"
    }
  },
  "external-navigation-modal": {
    "header": "Kumpirmahin ang Nabigasyon",
    "warning": "Magbubukas ka ng panibagong tab sa labas ng application. Pindutin ang Magpatuloy para magbukas ng panibagong tab o pindutin ang Kanselahin para bumalik sa application.",
    "cancel-button": "Kanselahin",
    "continue-button": "Magpatuloy"
  }
}
