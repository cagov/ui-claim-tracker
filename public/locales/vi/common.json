{
  "header": {
    "alt-image-cagov": "CA.gov",
    "alt-image-edd": "Sở Phát Triển Việc Làm (EDD) Tiểu Bang California",
    "edd-home": "Trang Chủ EDD",
    "help": "Trợ giúp",
    "logout": "Đăng Xuất",
    "uio-home": "Trang Chủ UI",
    "uio-certify": "Chứng Nhận",
    "uio-payments": "Các Khoản Thanh Toán",
    "uio-history": "Lịch sử",
    "uio-status": "Trạng thái",
    "uio-profile": "Hồ Sơ",
    "uio-inbox": "Hộp Thư Đến",
    "uio-contact": "Liên Hệ"
  },
  "title": "Công Cụ Theo Dõi Trạng Thái Hồ Sơ Yêu Cầu Thanh Toán",
  "welcome": "Công Cụ Theo Dõi Trạng Thái Hồ Sơ Yêu Cầu Thanh Toán",
  "errors": {
    "not-found": {
      "label": "Chúng tôi không thể tìm thấy trang đó.",
      "entry": "Liên kết quý vị nhấp vào có thể bị hỏng hoặc trang có thể đã bị xóa."
    },
    "button": "Trở Lại Trang Chủ UI",
    "server-error": {
      "label": "Đã xảy ra lỗi từ phía chúng tôi.",
      "entry": "Chúng tôi không thể thực hiện yêu cầu của quý vị."
    }
  },
  "maintenance": {
    "label": "Chúng Tôi Sẽ Quay Lại Ngay",
    "entry": "Trang này hiện không khả dụng trong quá trình bảo trì hệ thống.",
    "entry2": "Cảm ơn quý vị đã kiên nhẫn khi chúng tôi tiếp tục cải thiện dịch vụ của mình."
  },
  "claim-status": {
    "title": "Trạng thái Hồ Sơ Yêu Cầu Thanh Toán",
    "your-next-steps": "Bước Tiếp Theo Của Quý Vị",
    "edd-next-steps": "Bước Tiếp Theo EDD"
  },
  "claim-details": {
    "title": "Chi tiết Hồ Sơ Yêu Cầu Thanh Toán ",
    "benefit-year": "Năm Nhận Trợ Cấp",
    "program-type": "Loại Hồ Sơ Yêu Cầu Thanh Toán",
    "claim-balance": "Số Dư Trong Hồ Sơ Yêu Cầu Thanh Toán",
    "weekly-benefit-amount": "Số Tiền Trợ Cấp Hàng Tuần",
    "last-payment-issued": "Khoản Thanh Toán Cuối Được Cấp",
    "none": "Không Có",
    "extension-type": "Loại Gia Hạn"
  },
  "footer": {
    "toTop": "Trở Lại Đầu Trang",
    "contact": "Liên Hệ Với EDD",
    "conditionsOfUse": "Điều Kiện Sử Dụng",
    "privacyPolicy": "Chính Sách Bảo Mật",
    "accessibility": "Khả Năng Truy Cập",
    "copyright": "Copyright © 2021 State of California"
  },
  "timeout-modal": {
    "header": "Phiên Của Quý Vị Sẽ Kết Thúc Sớm",
    "warning": "Để bảo vệ thông tin của quý vị, quý vị sẽ bị đăng xuất sau {{count}} phút nếu quý vị không tiếp tục.",
    "warning_plural": "Để bảo vệ thông tin của quý vị, quý vị sẽ bị đăng xuất sau {{count}} phút nếu quý vị không tiếp tục.",
    "button": "Trở Lại Trang Chủ UI"
  },
  "time": {
    "between-range": ", trong khoảng {{range}} {{ampm}} Giờ Thái Bình Dương",
    "between-start-end": ", trong khoảng {{start.time}} {{start.ampm}} đến {{end.time}} {{end.ampm}} Giờ Thái Bình Dương",
    "am": "sáng",
    "pm": "chiều"
  },
  "urls": {
    "edd": {
      "ui-certify": "https://edd.ca.gov/Unemployment/certify.htm",
      "ui-claim-status": "https://www.edd.ca.gov/unemployment/claim-status.htm",
      "ui-reopen": "https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm",
      "ui-calculator": "https://edd.ca.gov/Unemployment/UI-Calculator.htm"
    },
    "hypertext": {
      "uio-homepage": "Trang chủ UI Online",
      "pending-payment": "Trạng thái Hồ Sơ Yêu Cầu Thanh Toán: Đang chờ thanh toán",
      "certify": "chứng nhận đủ điều kiện nhận trợ cấp",
      "reopen": "Mở Lại Hồ Sơ Yêu Cầu Thanh Toán",
      "reapply": "nộp lại đơn xin trợ cấp thất nghiệp",
      "enough-wages": "đủ lương"
    }
  },
  "external-navigation-modal": {
    "header": "Xác Nhận ĐIều Hướng",
    "warning": "Quý vị đang mở một tab mới bên ngoài ứng dụng. Chọn Tiếp Tục để mở tab mới hoặc chọn Hủy để quay lại ứng dụng.",
    "cancel-button": "Hủy",
    "continue-button": "Tiếp Tục"
  }
}
