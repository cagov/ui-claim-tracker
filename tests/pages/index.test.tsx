import renderer from 'react-test-renderer'
import { render, screen } from '@testing-library/react'
import Index from '../../pages/index'
import getScenarioContent from '../../utils/getScenarioContent'
import { ScenarioContent } from '../../types/common'

import { useRouter } from 'next/router'

jest.mock('next/router', () => ({
  __esModule: true,
  useRouter: jest.fn(),
}))

let scenarioContent: ScenarioContent

beforeAll(() => {
  const pendingDeterminationScenario = { pendingDetermination: ['temporary text'] }
  scenarioContent = getScenarioContent(pendingDeterminationScenario)
})

describe('Exemplar react-test-renderer Snapshot test', () => {
  it('renders homepage unchanged', () => {
    const mockRouter = {
      locale: 'en',
    }
    ;(useRouter as jest.Mock).mockReturnValue(mockRouter)

    const tree = renderer.create(<Index loading={false} scenarioContent={scenarioContent} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('Example react testing-library Test', () => {
  it('has our placeholder app', () => {
    render(<Index loading={false} scenarioContent={scenarioContent} />)
    expect(screen.queryByText('Claim Tracker')).toBeInTheDocument()
  })
})
