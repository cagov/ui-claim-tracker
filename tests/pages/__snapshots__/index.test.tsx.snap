// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exemplar react-test-renderer Snapshot test renders homepage unchanged 1`] = `
<div
  className="index container-fluid"
>
  <div
    className="fade alert alert-primary alert-dismissible show"
    fixed-top="true"
    role="alert"
  >
    <button
      className="close"
      onClick={[Function]}
      type="button"
    >
      <span
        aria-hidden="true"
      >
        ×
      </span>
      <span
        className="sr-only"
      >
        Close alert
      </span>
    </button>
    <div
      className="work-in-progress"
    >
      <div
        className="work-in-progress-content"
      >
        <span
          className="work-in-progress-title"
        >
          This page is a work in progress.
        </span>
         
        <span
          className="work-in-progress-message"
        >
          While we test this new part of UI Online, the information here may be limited for your claim. We're working to add information about common issues that lead to 
          <a
            href="https://navapbc.com"
          >
            pending payments
          </a>
          .
        </span>
      </div>
    </div>
  </div>
  <header
    className="header border-bottom border-secondary"
  >
    <nav
      className="justify-content-between navbar navbar-expand-lg navbar-dark"
      fixed-top="true"
    >
      <div
        className="container"
      >
        <a
          className="navbar-brand"
          href="https://ca.gov"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="CA.gov"
            height="34"
            src="/claimstatus/images/Ca-Gov-Logo-Gold.svg"
            width="46"
          />
        </a>
        <div
          className="navbar-nav"
          onKeyDown={[Function]}
        >
          <div
            aria-expanded={null}
            className="navbar-collapse collapse"
          >
            <a
              className="nav-link"
              data-rb-event-key="https://edd.ca.gov"
              href="https://edd.ca.gov"
              onClick={[Function]}
              onKeyDown={[Function]}
              rel="noopener noreferrer"
              target="_blank"
            >
              <span
                className="text"
              >
                EDD Home
              </span>
            </a>
          </div>
          <a
            className="nav-link"
            data-rb-event-key="https://askedd.edd.ca.gov/"
            href="https://askedd.edd.ca.gov/"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              className="text"
            >
              Help
            </span>
          </a>
          <a
            className="nav-link"
            data-rb-event-key="https://edd.ca.gov/login.htm"
            href="https://edd.ca.gov/login.htm"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              className="text"
            >
              Log out
            </span>
          </a>
        </div>
      </div>
    </nav>
    <nav
      className="justify-content-between navbar navbar-expand-lg navbar-light"
    >
      <div
        className="container"
      >
        <a
          className="navbar-brand"
          href="https://edd.ca.gov"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="EDD Employment Development Department State of California Home"
            className="d-inline-block align-top mr-5"
            height="60"
            src="/claimstatus/images/edd-logo-2-Color.svg"
            width="171"
          />
        </a>
        <button
          aria-controls="basic-navbar-nav"
          aria-label="Toggle navigation"
          className="navbar-toggler collapsed"
          onClick={[Function]}
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </button>
        <div
          aria-expanded={null}
          className="navbar-collapse collapse"
        >
          <div
            className="navbar-nav"
            onKeyDown={[Function]}
          >
            <a
              className="nav-link"
              data-rb-event-key="https://uio.edd.ca.gov"
              href="https://uio.edd.ca.gov"
              onClick={[Function]}
              onKeyDown={[Function]}
              rel="noopener noreferrer"
              target="_blank"
            >
              <span
                className="text"
              >
                UI Online Home
              </span>
            </a>
          </div>
        </div>
      </div>
    </nav>
  </header>
  <main
    className="main"
  >
    <div
      className="main-content container"
    >
      <h1
        className="title"
      >
        Claim Tracker
      </h1>
      <div
        className="language-switcher"
      >
        <span
          className="text"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          En español
        </span>
      </div>
      <div
        className="claim-section"
      >
        <div
          className="claim-status"
        >
          <h2>
            Claim Status
          </h2>
          <div
            className="pending-status"
          >
            <span
              className="text-line"
            >
              We need to confirm your eligibility for benefits.
            </span>
          </div>
          <div
            className="status-box"
          >
            <div
              className="topbar"
            >
              <h3
                className="next-steps"
              >
                Next Steps
              </h3>
            </div>
            <div
              className="explanation"
            >
              <div
                className="next-steps"
              >
                <ul>
                  <li
                    className="next-step"
                  >
                    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
                  </li>
                  <li
                    className="next-step"
                  >
                    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div
          className="claim-details container"
        >
          <h2>
            Claim Details
          </h2>
          <div
            className="claim-details-box"
          >
            <div
              className="row"
            >
              <div
                className="col-6"
              >
                <div
                  className="info"
                >
                  <div
                    className="info-label"
                  >
                    Program Type
                  </div>
                  <div
                    className="info-entry"
                  >
                    Unemployment Insurance (UI)
                  </div>
                </div>
                <div
                  className="info"
                >
                  <div
                    className="info-label"
                  >
                    Benefit Year
                  </div>
                  <div
                    className="info-entry"
                  >
                    3/21/2020 - 3/20/2021
                  </div>
                </div>
                <div
                  className="info"
                >
                  <div
                    className="info-label"
                  >
                    Claim Balance
                  </div>
                  <div
                    className="info-entry"
                  >
                    $0.00
                  </div>
                </div>
                <div
                  className="info"
                >
                  <div
                    className="info-label"
                  >
                    Weekly Benefit Amount
                  </div>
                  <div
                    className="info-entry"
                  >
                    $111.00
                  </div>
                </div>
                <div
                  className="info"
                >
                  <div
                    className="info-label"
                  >
                    Last Payment Issued
                  </div>
                  <div
                    className="info-entry"
                  >
                    4/29/2021
                  </div>
                </div>
              </div>
              <div
                className="col-6"
              >
                <div />
                <div
                  className="info"
                >
                  <div
                    className="info-label"
                  >
                    Extension Type
                  </div>
                  <div
                    className="info-entry"
                  >
                    Tier 2 Extension
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <nav
        className="justify-content-between navbar navbar-expand navbar-dark"
      >
        <div
          className="flex-column flex-sm-row navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            className="nav-link"
            data-rb-event-key="#top"
            href="#top"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Back to Top
          </a>
          <a
            className="nav-link"
            data-rb-event-key="https://edd.ca.gov/about_edd/contact_edd.htm"
            href="https://edd.ca.gov/about_edd/contact_edd.htm"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            target="_blank"
          >
            Contact EDD
          </a>
          <a
            className="nav-link"
            data-rb-event-key="https://edd.ca.gov/about_edd/conditions_of_use.htm"
            href="https://edd.ca.gov/about_edd/conditions_of_use.htm"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            target="_blank"
          >
            Conditions of Use
          </a>
          <a
            className="nav-link"
            data-rb-event-key="https://edd.ca.gov/about_edd/privacy_policy.htm"
            href="https://edd.ca.gov/about_edd/privacy_policy.htm"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            target="_blank"
          >
            Privacy Policy
          </a>
          <a
            className="nav-link"
            data-rb-event-key="https://edd.ca.gov/about_edd/accessibility.htm"
            href="https://edd.ca.gov/about_edd/accessibility.htm"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            target="_blank"
          >
            Accessibility
          </a>
        </div>
      </nav>
    </div>
    <div
      className="secondary-footer"
    >
      <div
        className="container"
      >
        <div
          className="copyright"
        >
          Copyright © 2021 State of California
        </div>
      </div>
    </div>
  </footer>
   
</div>
`;
